skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -0,0 +1,41 @@
+package com.RDS.skilltree.Endorsement;
+
+import jakarta.persistence.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/v1/endorsements")
+@Slf4j
+@RequiredArgsConstructor
+public class EndorsementController {
+    private final EndorsementService endorsementService;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<EndorsementResponse<EndorsementDTO>> getEndorsementById(@PathVariable(value = "id", required = true) String id){
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
```

Comment:
This we can also do by `@Produces(MediaType.APPLICATION_JSON)` this can be removed then.

Reply:
Implemented